src/components/Judging.js

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import { useState } from "react";
+import SelectSearch from "react-select-search";
+import Fuse from "fuse.js";
+
+import NonLiveDemoIndex from "../assets/non-live-index.svg";
+
+import Judges from "../data/judging/judges.json";
+import Projects from "../data/judging/projectData.json";
+import JudgeSchedules from "../data/judging/results.json";
+import ProjectSchedules from "../data/judging/projectResults.json";
+
+const TIME_SLOTS = [
+  "10:30",
```

Comment:
aren't we doing 8 min slots?

Reply:
just added screenshots! idk about the time slots, @zefwang ?

Reply:
we are doing 8 minute slots, I will fix this